build.gradle

```
@@ -10,7 +10,7 @@ plugins {
 }
 
 group 'com.github.hmcts'
-version '1.0.16.1'
+version '1.0.16.2'
```

Could we go back to using semver for this? MAJOR.MINOR.PATCH
```suggestion
version '1.0.17'
```